gufe/tokenization.py

```diff
@@ -199,7 +199,7 @@ def defaults(cls):
         """Dict of default key-value pairs for this `GufeTokenizable` object.
 
         These defaults are stripped from the dict form of this object produced
-        with `to_dict(include_defaults=False) where default values are present.
+        with `to_dict(include_defaults=False)` where default values are present.
```

Comment:
```suggestion
        with ``to_dict(include_defaults=False)`` where default values are present.
```

whoever wrote these docs forgot that docs should be in RST.

Reply:
Ok so I wasn't sure about this one because the NumPy docstring spec has this insane thing where you're supposed to put variable names in single backticks, so I never know whether to correct it. This particular case is not a variable so it should definitely be in double backticks... But this kind of thing is alllll over the docstrings. I'll quickly do a grep to see if I can find them all!